src/main/java/Kvothe.java

```diff
@@ -0,0 +1,204 @@
+import java.util.Scanner;
+
+public class Kvothe {
+
+    private static final int MAXTASKS = 100;
+    private static Task[] tasks = new Task[MAXTASKS];
+    private static int tasksIndex = 0;
+
+    public static void main(String[] args) {
+        String greeting = "Hello! I'm Kvothe.\n"
+                + "\t\tWhat can I do for you?\n";
+
+        String bye = "Bye. Hope to see you again soon!";
+
+        echo(greeting);
+
+        interact();
+
+        echo(bye);
+    }
+
+
+    private static void echo(String line, boolean isTop, boolean isBottom) {
+        if (isTop) {
+            System.out.println("\t\t____________________________________________________________");
+        }
+
+        System.out.println("\t\t" + line);
+
+        if (isBottom) {
+            System.out.println("\t\t____________________________________________________________");
+        }
+
+    }
+
+    private static void echo(String line) {
+        System.out.println("\t\t____________________________________________________________");
+        System.out.println("\t\t" + line);
+        System.out.println("\t\t____________________________________________________________");
+
+    }
+
+    //TODO: move to Task class?
+    //TODO: Check that the arguments are the correct ones
+    private static String[] parseLine(String line, String[] args) {
+        String[] lineWords = line.split(" ");
+        String[] newArgs = new String[args.length + 1]; // description + args
+
+        int lastWordIndex = 1;
+        int argsIndex = 0;
+
+        // Get the description
+        String description = "";
+        while (lastWordIndex < lineWords.length && !lineWords[lastWordIndex].startsWith("/")) {
```

Comment:
I think this expression is a little complicated. I think it will be more intuitive if the code is broken down into more steps.